scripts/build/Platform/Linux/build_config.json

```diff
@@ -132,6 +132,35 @@
       "ASSET_PROCESSOR_PLATFORMS": "linux,server"
     }
   },
+  "awsi_test_profile_pipe": {
+    "TAGS": [
+      "nightly-incremental",
+      "nightly-clean"
+    ],
+    "steps": [
+      "awsi_deployment",
+      "awsi_test_profile",
+      "awsi_destruction"
+    ]
+  },
+  "awsi_test_profile": {
+    "TAGS": [
+        "weekly-build-metrics"
+    ],
+    "PIPELINE_ENV": {
+      "NONBLOCKING_STEP": "True"
+    },
+    "COMMAND": "build_test_linux.sh",
+    "PARAMETERS": {
+      "CONFIGURATION": "profile",
+      "OUTPUT_DIRECTORY": "build\\linux",
+      "CMAKE_OPTIONS": "-G 'Ninja Multi-Config' -DLY_PARALLEL_LINK_JOBS=4",
```

Comment:
note: 4 should ideally be pulled in from config. I would assume the build infra to setting this depending on hardware - container vs Jenkins vs VM etc).

Reply:
Yeah, the CMake options are copied from other Jenkins jobs defined for Linux. We can send it as a feature request to infra.

Reply:
To be fair, this file is a config file, unless you mean build environment. We could pull in from a global env var, but this particular attribute shouldn't vary depending on the build environment (one use is to meant to limit memory usage during linking)